src/main/resources/db/migration/V9__ekstern_id.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE iverksett
+    ADD COLUMN ekstern_id BIGINT NOT NULL;
```

Comment:
Her må man splitte opp denne med uten `NOT NULL` og legge på det etter att du har oppdatert iverksett med `ekstern_id`

Reply:
Godt observert 👍  Legger til NOT NULL constraint til slutt